packages/stores/types.ts

```diff
@@ -18,6 +18,7 @@ import type {
 import type { Moment } from 'moment';
 import type { RouteComponentProps } from 'react-router';
 import type { ExchangeRatesStore, FeatureFlagsStore } from './src/stores';
+import { TContractStore as ContractStore } from '../shared/src/utils/contract';
```

Comment:
@henry-deriv Please **DO NOT** improve cross packages :sad-cat:

Reply:
@farzin-deriv removed